index.js

```diff
@@ -0,0 +1,10 @@
+const PullRequest = require('./src/github').PullRequest;
+
+module.exports = app => {
+  app.on(['pull_request.opened'], pullRequest)
```

Comment:
What about other events? According to [this](https://github.com/ampproject/amphtml/blob/ece2fa3ab444935fa5101672735e5b39e7ff4d18/contributing/CODE_OWNERSHIP.md) in-flight PR to document the owners bot, we should have:

- Creation of a pull request
- New commit pushed to a pull request
- Comment posted on a pull request
- Pull request reviewed

Reply:
i believe the only other event we need to react to is synchronize (updates to PR)

the old code needed to monitor comments because it used commends for approving/reviews. I'll update the document. thanks for pointing it out